src/modules/job-manager/jobtap.c

```diff
@@ -2683,6 +2683,26 @@ int jobtap_validate_updates (struct jobtap *jobtap,
     return rc;
 }
 
+int flux_jobtap_call (flux_plugin_t *p,
+                      flux_jobid_t id,
+                      const char *topic,
+                      flux_plugin_arg_t *args)
+{
+    struct jobtap *jobtap;
+    struct job *job;
+
+    if (p == NULL
+        || !(jobtap = flux_plugin_aux_get (p, "flux::jobtap"))
+        || !jobtap->ctx
+        || !args) {
+        errno = EINVAL;
+        return -1;
+    }
+    if (!(job = jobtap_lookup_jobid (p, id)))
+        return -1;
+    return jobtap_stack_call (jobtap, jobtap->plugins, job, topic, args);
```

Comment:
I would imagine calling the documented jobtap topics would lead to sadness here.  Any way to prevent that?  Maybe all topics used here should be required to have a `user.*` prefix or something?

Reply:
I'm on the fence. We don't require the same for the equivalent shell functions, but I can see your point and the price of failure is higher in this case. However, I'm not sure `user.` makes sense here because this call could be implemented by system plugins. I think all the documented jobtap topics start with `job.` so maybe disallowing a call of `job.*` here would be sufficient?

Reply:
Works for me!

Reply:
Ok, I pushed an update that doesn't allow `flux_jobtap_call()` to be used with topic strings starting with `job.`.
The job manager also uses `plugin.query` and `conf.update`, however those are not prevented here because perhaps someone will think of a reason to call them from another plugin, and the chances of misuse are very low.